src/collections/Table/Table.js

```diff
@@ -84,7 +84,7 @@ function Table(props) {
     <ElementType {...rest} className={classes}>
       {headerRow && <TableHeader>{TableRow.create(headerRow, { cellAs: 'th' })}</TableHeader>}
       <TableBody>
-        {renderBodyRow && _.map(tableData, (data, index) => TableRow.create(renderBodyRow(data, index)))}
+        {renderBodyRow && TableRow.create(_.map(tableData, (data, index) => renderBodyRow(data, index)))}
```

Comment:
I think this change is incorrect. This should map over each item in the `tableData` array and create a `TableRow` from each one. This change would map over each item in the `tableData` array and create one giant `TableRow`.